etna/packages/magby/magby/Magby.py

```diff
@@ -74,19 +74,24 @@ def retrieve(self,
     def query(self,
               projectName: str,
               queryTerms: List,
+              format: str = "json",
               **kwargs) -> Dict:
         '''
         Performs an expressive query to Magma.
         :param projectName: str. Project name in Magma
         :param queryTerms: List[str]. Query terms to magma. See Magma documentation
+        :param formt: str. "json" or "tsv". Adjusts the format of the returned data. See Magma documentation
```

Comment:
@coleshaw, please correct if needed

Reply:
"json" won't actually do anything. The backend will currently default to JSON response unless `format=tsv` is passed in.

Reply:
gotcha!  Adjusted here.  We should add this and the distinct change to the mountetna.github.io query docs.

Reply:
Oh thanks, good reminder! I'll put in a PR to update the query docs for the wiki.